test-data-server/src/services/single.service.ts

```diff
@@ -605,13 +605,13 @@ export class SingleData implements IDatabase {
                                 nickname: acc.account?.plans[i]?.nickname,
                                 servicePointIds: []
                             }
-                            if (acc.account?.openStatus == null || acc.account?.openStatus?.toUpperCase() == openStatus?.toUpperCase())
+                            if (acc.account?.openStatus == "OPEN")
```

Comment:
This if is incorrect. The planOverview field should be assigned unconditionally.

Reply:
Modified logic to include planOverview if it does exists